src/js/index.js

```diff
@@ -0,0 +1,52 @@
+import Todo from './models/todo';
+import TaskManager from './controllers/taskmanager';
+import '../css/style.css';
+
+const taskManager = new TaskManager();
+
+const demoData = () => {
+  if (taskManager.taskList.length === 0) {
+    const singleTask = new Todo('Do action one', false, 1);
+    taskManager.addTask(singleTask);
+
+    const singleTask1 = new Todo('Do action two', false, 2);
+    taskManager.addTask(singleTask1);
+
+    const singleTask2 = new Todo('Do action three', true, 0);
+    taskManager.addTask(singleTask2);
```

Comment:
- I think it's better to create an array of objects (description, completed, and index). It's a project requirement.